core/src/test/java/hudson/model/ListViewTest.java

```diff
@@ -24,6 +25,7 @@
     
     private interface ItemGroupOfNonTopLevelItem extends TopLevelItem, ItemGroup<Item> {}
 
+    @Ignore("TODO I am not smart enough to figure out what PowerMock is actually doing; whatever this was testing, better move to the test module and use JenkinsRule")
```

Comment:
Uh, yeah. Apply the transformation in this test suite and both cases fail with an opaque `NullPointerException`. Leave the suite alone and the other case passes but this one fails with a comparison error—unless you run just this case by itself, in which case it passes.